image.yaml

```diff
@@ -2,33 +2,33 @@ schema_version: 1
 
 name: "redhat-sso-7/sso72"
 description: "Red Hat Single Sign-On 7.2 container image"
-version: "7.2.2"
-from: "jboss-eap-7/eap71:7.1.2-2"
+version: "7.2.3"
+from: "jboss-eap-7/eap71:7.1.3-3"
```

Comment:
Is it better to use `:7.1.3`, to avoid having to update this file, e.g. in the event the base eap image is rebuilt to address a cve?

Reply:
Are you referring to Freshmaker? If yes then it doesn't matter really. It matters in case of manual rebuilds for a product CVE though.

Reply:
Not freshmaker.  Manual rebuilds would include sprint builds too.

Reply:
@rcernich I personally like better the approach explicitly to specify image tag, the RH-SSO image should be rebuilt against (see the recent case of EAP 7.1.3-2 vs EAP 7.1.3-3 rebuild for example). Also, when editing this file to sync the changes post release, we would need to update image version and Keycloak server overlay zip version anyway (thus making one more line change doesn't look like too much difference to me).

I am trying to keep this file updated, so the image is built properly also in case @ASzc or me are on PTO, and the image is built by someone else (from the experience it's enough complexity for them to explain on how to build the image even without explaining how to check, which EAP image the RH-SSO image was built against, and ensure it's the proper one). This approach is trying to simplify the build process for them for now a bit.